### ISSUE TITLE
Individuals: Bool header filter icons are mixed up

### ISSUE THREAD

**Issue by almanilsson** [hannamlmv/tseda#58]
The icons for the header filter boolean options are mixed up. See figures below. This is a problem with panel and we should probably report the issue to them, but perhaps we can fix it ourselves. 

"False" shows all
![image](https://github.com/user-attachments/assets/5e21a2c7-f149-4156-a7bb-5cabbd3f3e42)

"True" shows selected
![image](https://github.com/user-attachments/assets/b82e0d84-fcc9-4a33-b53a-19eda49aaa61)

And having neither shows the ones that are not selected 
![image](https://github.com/user-attachments/assets/c0de995c-3d25-4391-937a-7514c0bb361c)

In summary False == all, True == True and all == False

**Comment by almanilsson** [comment]
This bug is present even on their example on their documentation website. See "Client-side filtering" on this page https://panel.holoviz.org/reference/widgets/Tabulator.html

**Comment by almanilsson** [comment]
This is a Panel issue so we wont fix it
